src/components/Button/lib.js

```diff
@@ -11,6 +11,7 @@ export const availableVariants = [
   'inverted-link',
   'primary-link',
   'caution',
+  'ghost',
```

Comment:
we removed the `ghost` class, it's now called `tertiary`

Reply:
Thank you for letting me know @lisilinhart. I removed 'ghost' from Button/lib.js and updated the card component to use the `tertiary` class instead.